halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/metricStores/datadog/DatadogStore.java

```diff
@@ -42,5 +45,5 @@ public String getNodeName() {
   private String appKey;
 
   @JsonProperty("tags")
-  private String tags;
+  private List<String> tags = new ArrayList<>();
```

Comment:
Does halyard complain during an upgrade that the schema changed?

Reply:
Don't understand what you mean by upgrade exactly, but I rebuilt the Halyard Binary and tested the changes locally to see if the Halconfig was correctly generated. I didn't see any errors or warnings with the changes.

Reply:
If your halconfig has `tags: <string>`, and you upgrade to include this patch, does halyard complain about deserialization failing because of a mismatched type?

Reply:
The Halyard master  branch doesn't have `tags` as a [member variable](https://github.com/spinnaker/halyard/blob/master/halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/metricStores/datadog/DatadogStore.java#L43). The Github UI seems to have messed that up 

If you click on the `Files Changed` tab then you won't see the redaction of the `tags` member.

Reply:
Ah got it, looks good to me!